specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/schema/kafka.schema.patch.json

```diff
@@ -78,6 +78,166 @@
                                         {
                                             "type": "string",
                                             "enum": [ "none", "json_patch" ]
+                                        },
+                                        "key":
+                                        {
+                                            "type": "object",
+                                            "additionalProperties": false,
+                                            "properties":
+                                            {
+                                                "type":
+                                                {
+                                                    "title": "Type",
+                                                    "type": "string",
+                                                    "enum":
+                                                    [
+                                                        "byte_buffer",
+                                                        "bytes",
```

Comment:
What is the difference in validation between `byte_buffer` and `bytes`?
(also in duplicated enum below)

Reply:
I don’t think we need these for validator. byte_buffer is just the bytes wrapped to give an instance of byte_buffer.